setup.py

```diff
@@ -97,7 +97,8 @@ def read_reqs(req_file):
     install_requires=install_requires,
     entry_points="""
       [console_scripts]
-      elastic-ingest = connectors.cli:main
+      elastic-ingest = connectors.old_cli:main
```

Comment:
nit - we should probably have a better name for this than `old_cli`. Like `shell` or `commands`. Or we can choose to roll its functionality into the `connectors_cli` if we're worried there will be too much confusion in having two "command line" sets of features.

Reply:
Good catch 👍🏻